src/app/zap-templates/zcl/data-model/chip/barrier-control-cluster.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="Closures"/>
+
+  <enum name="BarrierControlMovingState" type="ENUM8">
+    <cluster code="0x0103"/>
```

Comment:
Please don't expose non-spec names like this one (which was not exposed as part of this cluster before this PR).

Really, we should not have been including this cluster at all, since it's not in the Matter spec.  But we should not expose any more of it until there is an actual spec and we make the XML reflect that spec.

Reply:
Btw we will likely discard Barrier Control for Matter and even sunset it in Zigbee

Reply:
Ok. So I did things a bit differently. I moved all the barrier control related definitions into a new `draft` folder in `src/app/zap-templates/zcl/data-model/`. So there is a single place of trust for the definition, and if needed, we could just remove it when needed.

I also did not update the definition, so nothing is exposed, I did just shuffle things around.

Reply:
> Btw we will likely discard Barrier Control for Matter and even sunset it in Zigbee

I have no problem removing it completely too ;)